src/main/java/bwcrossmod/galacticgreg/MTEVoidMiners.java

```diff
@@ -82,28 +341,145 @@ public IMetaTileEntity newMetaEntity(IGregTechTileEntity iGregTechTileEntity) {
+                    "Can be supplied with 2L/s of Neon(x4), Krypton(x8), Xenon(x16) or Oganesson(x64) for higher outputs.")
+                .addInfo(
+                    "Will output " + 2 * this.TIER_MULTIPLIER
+                        + " Ores per Second depending on the Dimension it is build in")
+                .addInfo("Put the Ore into the input bus to set the Whitelist/Blacklist")
+                .addInfo("Use a screwdriver to toggle Whitelist/Blacklist")
+                .addInfo(
+                    "Blacklist or non Whitelist Ore will be " + EnumChatFormatting.DARK_RED
+                        + "VOIDED"
+                        + EnumChatFormatting.RESET
+                        + ".")
+                .beginStructureBlock(9, 16, 9, false)
+                .addStructureInfo(
+                    EnumChatFormatting.GOLD + "47x" + EnumChatFormatting.GRAY + " Mining Neutronium Casing")
+                .addStructureInfo(
+                    EnumChatFormatting.GOLD + "8x" + EnumChatFormatting.GRAY + " Black Plutonium Item Pipe Casing")
+                .addStructureInfo(EnumChatFormatting.GOLD + "72x" + EnumChatFormatting.GRAY + " Adamantium Frame Box")
+                .addStructureInfo(EnumChatFormatting.GOLD + "20x" + EnumChatFormatting.GRAY + " Bolted Iridium Casing")
+                .addStructureInfo(
+                    EnumChatFormatting.GOLD + "36x" + EnumChatFormatting.GRAY + " Rebolted Iridium Casing")
+                .addEnergyHatch(VN[this.getMinTier()] + "+, Any base casing")
+                .addMaintenanceHatch("Any base casing")
+                .addInputBus("Mining Pipes or Ores, optional, any base casing")
+                .addInputHatch("Optional noble gas, any base casing")
+                .addOutputBus("Any base casing")
+                .toolTipFinisher();
+            return tt;
+        }
+
+        public void construct(ItemStack stackSize, boolean hintsOnly) {
+            buildPiece(STRUCTURE_PIECE_MAIN, stackSize, hintsOnly, 4, 13, 2);
+        }
+
+        public int survivalConstruct(ItemStack stackSize, int elementBudget, ISurvivalBuildEnvironment env) {
+            if (mMachine) return -1;
+            return survivialBuildPiece(STRUCTURE_PIECE_MAIN, stackSize, 4, 13, 2, elementBudget, env, false, true);
         }
 
         @Override
-        protected Materials getFrameMaterial() {
-            return Materials.Neutronium;
+        public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack aStack) {
+            return (checkPiece(STRUCTURE_PIECE_MAIN, 4, 13, 2)
+                // old structure (to be deprecated on future version)
+                || (checkPiece(STRUCTURE_PIECE_OLD, 1, 6, 0))) && checkHatches()
+                && GTUtility.getTier(getMaxInputVoltage()) >= getMinTier()
+                && mMaintenanceHatches.size() == 1;
+        }
+
+        public VMUV(String aName, int tier) {
+            super(aName, tier);
+            casingTextureIndex = 178;
```

Comment:
nitpick: same here (along with getControllerTextureIndex below)

```suggestion
            casingTextureIndex = Casings.MiningNeutroniumCasing.getTextureId();
```